### PR TITLE
Added subfolders and files for user guide

### PR DESCRIPTION
**Pull request by sjib**
dito folders data-import / export - now ready for editing by QGEP team members

### CODE REVIEW

**Comment by sjib** [comment]
@m-kuhn Can you include this into the master of QGEP/docs so that others can start working?

File: en/admin-guide/data-import/index.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.. _Admin Guide:
+
+Data import
+====================
+
+This represents a guide on how to import data into QGEP.
+* INTERLIS
```
Comment (m-kuhn) [inline en/admin-guide/data-import/index.rst]: I think this needs an extra newline before lists. See the "View" link on top for the preview

File: en/admin-guide/data-export/index.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+^^^^^^^^^^^^^^^^^
+
+* List
+  * Subpoint
+  
+Link:
+add `link <http://www.postgresql.org/docs/current/static/libpq-pgpass.html>`_
+
+Add some screen shots 
+.. figure:: images/001_addplugin.jpeg
```
Comment (m-kuhn) [inline en/admin-guide/data-export/index.rst]: I think this needs an extra newline before

    .. figure:: to work

**Comment by sjib** [comment]
@m-kuhn So jetzt sollte es besser sein.

**Comment by m-kuhn** [comment]
danke Stefan

**Comment by sjib** [comment]
What happend to the Admin Guide and all the things you had allready on? http://qgep.github.io/docs/

**Pull request merged**